src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -115,9 +113,17 @@ func Init() (*Config, error) {
 			return
 		},
 	}
-	initial, err := m.Start()
+	initial, err := m.Init()
 	var cfg *Config
 	if err == nil {
```

Comment:
Here it would be a good place to declare the `Once` I mention in the next comment.
```go
var once sync.Once
```